jmx/jmx.go

```diff
@@ -341,6 +341,7 @@ func Query(objectPattern string, timeoutMillis int) (result map[string]interface
 
 // receiveResult checks for channels to receive result from nrjmx command.
 func receiveResult(lineC chan []byte, queryErrC chan error, cancelFn context.CancelFunc, objectPattern string, timeout time.Duration) (result map[string]interface{}, err error) {
+	defer logAvailableWarnings()
```

Comment:
do we still need this?

Reply:
yes, we want to print all warnings even if data was processes

Reply:
I moved global var as a param. We can create bool variable for the loop and move it after that, but it will increase complexity IMHO